connect/eaas/core/dataclasses.py

```diff
@@ -56,32 +56,31 @@ class MessageType:
 class TaskOptions(BaseModel):
     task_id: str
     task_category: str
-    result: Optional[str]
-    countdown: int = 0
-    runtime: float = 0.0
-    output: Optional[str]
     correlation_id: Optional[str]
     reply_to: Optional[str]
 
 
+class TaskOutput(BaseModel):
+    result: str
+    data: Optional[Any]
```

Comment:
`data` field is for previous `output`?
Moreover, if EaaS we use slicing for it, so `Optional[Any]` can potentially cause errors.

Reply:
Saw, that `error` is for `output`. For what than data is needed?

Reply:
No, data is for interactive tasks, in general it is a dictionary but can be any additional output data.